aptos-move/aptos-aggregator/src/tests/types.rs

```diff
@@ -100,4 +102,25 @@ impl TDelayedFieldView for FakeAggregatorView {
         *counter += 1;
         id
     }
+
+    fn validate_and_convert_delayed_field_id(&self, id: u64) -> Result<Self::Identifier, PanicError> {
```

Comment:
I don't think this is used in tests? e.g. `set_from_aggregator_id` uses a delayed id already